frontend/src/components/manageUsers/PageHeader.tsx

```diff
@@ -0,0 +1,3 @@
+export const PageHeader = ({ pageHeader }: { pageHeader: string }) => {
+  return <h1 className="font-sans-2xl">{pageHeader}</h1>;
```

Comment:
can we add the org name back? you were right the first time, it just wasn't anywhere in the ticket